addons/hr_recruitment/tests/test_recruitment_process.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import common
+from odoo.modules.module import get_module_resource
+
+
+class HrRecruitmentTestCase(common.TransactionCase):
+
+    def setUp(self):
```

Comment:
`setUp` is important for mixins or when there are multiple tests using the same resource set up. Here it is completely useless and just increases noise.